tests/test_modeling_common.py

```diff
@@ -66,12 +66,16 @@ class ModelTesterMixin:
 
     def _prepare_for_class(self, inputs_dict, model_class):
         if model_class in MODEL_FOR_MULTIPLE_CHOICE_MAPPING.values():
+            if "lengths" in inputs_dict:
```

Comment:
@sgugger without this, [this](https://github.com/huggingface/transformers/blob/master/src/transformers/modeling_xlm.py#L456) assertion fails

Reply:
I think this is the sign the lengths should be flattened like the input_ids if present.